package/gluon-mesh-vpn-tunneldigger/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/lua
+
+function restart_tunneldigger()
+	io.popen('logger -t tunneldigger-watchdog "Restarting Tunneldigger."')
+	os.execute("/etc/init.d/tunneldigger restart")
+end
+
+function read_pid_file()
+	local pid_file = io.open("/var/run/tunneldigger.mesh-vpn.pid", "r")
+	if not pid_file then
+		return nil
+	end
+	pid = pid_file:read("*l") 
```

Comment:
There should be no reason to declare `pid` as a global variable here. So declare it as a local variable.